templates/vhosts.conf.j2

```diff
@@ -15,29 +14,17 @@
 {% if vhost.redirect_ssl | default(false) %}
   Redirect permanent / https://{{ apache_listen_ip }}:{{ apache_listen_port_ssl }}
 {% endif %}
-{% if vhost.serveradmin is defined %}
```

Comment:
Why remove the ServerAdmin and extra parameter statements?

Reply:
No specific reason, to keep it simple. Just added them again.